src/cpp/sensitivity/AnalysisFunctions.cpp

```diff
@@ -31,21 +27,29 @@ SolverAbstract::Ptr get_sensitivity_problem(
     const SensitivityInputData input_data, const std::string& candidate_name,
     SensitivityPbType type) {
   unsigned int nb_candidates = input_data.name_to_id.size();
+  SolverAbstract::Ptr sensitivity_problem;
   switch (type) {
     case SensitivityPbType::CAPEX: {
       ProblemModifierCapex pb_modifier(input_data.epsilon, input_data.best_ub,
                                        input_data.last_master);
-      return pb_modifier.changeProblem(nb_candidates);
+      sensitivity_problem = pb_modifier.changeProblem(nb_candidates);
+      break;
     }
     case SensitivityPbType::PROJECTION: {
       ProblemModifierProjection pb_modifier(
           input_data.epsilon, input_data.best_ub, input_data.last_master,
           input_data.name_to_id.at(candidate_name), candidate_name);
-      return pb_modifier.changeProblem(nb_candidates);
+      sensitivity_problem = pb_modifier.changeProblem(nb_candidates);
+      break;
     }
 
     default:
       std::cerr << "Unrecognized Sensitivity Problem type" << std::endl;
       return nullptr;
   }
+
+  if (std::ifstream basis_file(input_data.basis_file_path); basis_file.good()) {
```

Comment:
If the file is not good nothing happens. Should we at least warn users or developers with log ?

Reply:
Yes, we can add a warning